packages/@aws-cdk/aws-applicationautoscaling/test/test.cron.ts

```diff
@@ -26,4 +26,22 @@ export = {
     }, /Duration cannot be 0/);
     test.done();
   },
+
+  'rate can be token'(test: Test) {
+    const stack = new Stack();
+    const invokeScheduleInMinutes = new CfnParameter(stack, 'invokeScheduleInMinutes', {
+      type: 'Number',
+      description: 'The minutes between scheduled invocations.',
+      default: 5,
+    });
```

Comment:
If you switch to using the `Lazy` class (which also uses the same token mechanism), it would be easier to test. Something like this -

```ts
const lazyDuration = Duration.minutes(Lazy.number({ produce: () => 5 }));
const rate = appscaling.Schedule.rate(lazyDuration)
test.equal('rate(<token> minutes)', rate);
test.equal('rate(5 minutes)', stack.resolve(rate));

Reply:
Sure, will give it a try.

Reply:
@nija-at Are you suggesting me to modify in all the testcases where I have used cfnParameters or few?